src/main/java/TypewiseAlert/Alerter/EmailAlerter.java

```diff
@@ -0,0 +1,15 @@
+package main.java.TypewiseAlert.Alerter;
+
+import main.java.TypewiseAlert.BreachType;
+
+public class EmailAlerter implements IAlerter

```

Comment:
EmailAlerter Incomplete

Reply:
Hi Sir,
I am not clear with this comment. Could you please help me to understand.

Reply:
understand  the requirement ...EmailAlerter further need to action based on breachType(Low or High)